tcms/static/js/utils.js

```diff
@@ -163,3 +163,34 @@ function treeViewBind() {
       }
     });
 }
+
+function filterTestCasesByProperty(testCases, property, value) {
+    const filterValue = value;
+    const filterBy = property;
+
+    for (let id in testCases) {
+        if (!testCases.hasOwnProperty(id)) {
+            break;
+        }
+
+        let tc = testCases[id];
+
+        // we delete all text from the input
+        if (filterValue.trim().length === 0) {
+            $(`[data-testcase-pk=${id}]`).show();
+            continue;
+        }
+
+        // check if testcase has such property
+        if (tc[filterBy] === undefined) {
```

Comment:
If a property exists and its null I want to hide the testcases. For example default_tester.

Reply:
sure. I am not proposing different behavior, just different style. in the common case the result of `!something` and `something === undefined` should be the same, unless `null` means something else than `undefined` in this context, which it should not have.

Reply:
actually I think I get it. but then we don't need this check at all, because of the check on line 190. 
```javascript
        // actual filtering, the property is null or does not contains the string
        if (!tc[filterBy] || tc[filterBy].toLowerCase().indexOf(filterValue) === -1) {
```
this will handle both `null` and `undefined` so no need to do it twice